src/repository/gorm2/v2_game_play_log_test.go

```diff
@@ -739,14 +981,40 @@ func TestGetEditionPlayStats(t *testing.T) {
 			stats, err := gamePlayLogRepository.GetEditionPlayStats(ctx, testCase.editionID, testCase.start, testCase.end)
 
 			if testCase.isErr {
-				if testCase.err == nil {
-					assert.Error(t, err)
-				} else {
-					assert.ErrorIs(t, err, testCase.err)
+				assert.Error(t, err)
+				if testCase.err != nil {
+					assert.Equal(t, testCase.err, err)
 				}
-			} else {
-				assert.NoError(t, err)
-				assert.Equal(t, testCase.expectedStats, stats)
+				return
+			}
+
+			require.NoError(t, err)
```

Comment:
テストの結果としてエラー出ないことを知りたいので、`assert`にしてください。

```suggestion
			assert.NoError(t, err)
```